docker_entrypoint.sh

```diff
@@ -53,19 +53,34 @@ server {
     text/javascript
     text/plain
     text/xml;
-    listen 3443 ssl;
+    listen 3443 ssl http2;
     listen 8080;
     ssl_certificate /mnt/cert/main.cert.pem;
     ssl_certificate_key /mnt/cert/main.key.pem;
     server_name  localhost;
+    client_max_body_size 128M;
 
     location / {
+        proxy_set_header Host $host;
```

Comment:
I added these nginx.conf settings at the suggestion of the Vaultwarden team:
https://vaultwarden.discourse.group/t/cant-upload-attatchments-to-vaultwarden/2704